Header header

# x
float32 rawVelX
float32 velX
float32 crrPosXNonBias
float32 crrVelXNonBias
float32 crrPosXBias
float32 crrVelXBias
float32 crrXBias
float32 emptyX1
float32 emptyX2


# y
float32 rawVelY
float32 velY
float32 crrPosYNonBias
float32 crrVelYNonBias
float32 crrPosYBias
float32 crrVelYBias
float32 crrYBias
float32 emptyY1
float32 emptyY2

# z
float32 rawPosZ
float32 PosZ
float32 rawVelZ
float32 crrPosZNonBias
float32 crrVelZNonBias
float32 crrPosZBias
float32 crrVelZBias
float32 crrZBias
float32 emptyZ1
float32 emptyZ2
